.eslintrc.js

```diff
@@ -198,6 +199,19 @@ module.exports = {
         pathGroupsExcludedImportTypes: [],
       },
     ],
+    "jest/consistent-test-it": ["error", { fn: "it" }],
```

Comment:
Explanations for all these rules can be found here: https://github.com/jest-community/eslint-plugin-jest